src/components/ConnectWalletBanner.tsx

```diff
@@ -14,6 +14,7 @@ export const Wrapper = styled.div`
   border-radius: 0.6rem;
   width: 100%;
   min-height: 35rem;
+  min-width: 55rem;
```

Comment:
do we have a variable for min-width stuff?

Reply:
not to my knowledge